lookout/style/format/rules.py

```diff
@@ -220,6 +220,7 @@ def fit(self, X: numpy.ndarray, y: numpy.ndarray) -> "TrainableRules":
         :param y: input labels - the same length as X.
         :return: self
         """
+        self._log.debug("TrainableRules.fit(...)")
```

Comment:
Can you please rewrite as `"fitting rules with config %s" % self.origin_config["trainable_rules"]` to be more consistent with our other logs and give a bit more info.

Reply:
I think it is better to have something like
```
self._log.debug("fitting rules with params %s", self.get_params(False))
```
because we use only this params  from config in fit().

Reply:
I think both are pretty similar no? Since with `origin_config["trainable_rules"]` we only pick the section about trainable rules. But yeah both are fine for me.

Reply:
mix up things, you are right.